CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ It diverges in the following ways:
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html) for bug fix releases.
 
 ## Unreleased
+### Added
+- Add support for Amazon Device Messaging (#2528)
```

Comment:
nit: remove "Add". The CHANGELOG is supposed to read like, "Added ... Support for Amazon Device Messaging".

Also, this isn't user facing yet so I'd preface with, "(disabled by default) Support for Amazon..."